<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>anitools::widgets::ColorPicker</class>
 <widget class="QWidget" name="anitools::widgets::ColorPicker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Color Picker</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>256</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>256</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="anitools::widgets::ColorSlider" name="mSliderBig" native="true">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>256</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="mButtonRed">
           <property name="text">
            <string>R</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="anitools::widgets::ColorSlider" name="mSliderRed" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinRed">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="mButtonGreen">
           <property name="text">
            <string>G</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="anitools::widgets::ColorSlider" name="mSliderGreen" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinGreen">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="mButtonBlue">
           <property name="text">
            <string>B</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="anitools::widgets::ColorSlider" name="mSliderBlue" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinBlue">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="mButtonHue1">
           <property name="text">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="anitools::widgets::ColorSlider" name="mSliderHue1" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinHue1">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="mButtonSaturation1">
           <property name="text">
            <string>S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="anitools::widgets::ColorSlider" name="mSliderSaturation1" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinSaturation1">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="mButtonValue">
           <property name="text">
            <string>V</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="anitools::widgets::ColorSlider" name="mSliderValue" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinValue">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="mButtonHue2">
           <property name="text">
            <string>H</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="anitools::widgets::ColorSlider" name="mSliderHue2" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinHue2">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="mButtonSaturation2">
           <property name="text">
            <string>S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="anitools::widgets::ColorSlider" name="mSliderSaturation2" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinSaturation2">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1,0">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="mButtonLightness">
           <property name="text">
            <string>L</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="anitools::widgets::ColorSlider" name="mSliderLightness" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>10</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="mSpinLightness">
           <property name="maximum">
            <number>255</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>anitools::widgets::ColorSlider</class>
   <extends>QWidget</extends>
   <header>colorslider.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
